PYME/recipes/measurement.py

```diff
@@ -178,8 +178,29 @@ def execute(self, namespace):
 
 @register_module('FitPoints')
 class FitPoints(ModuleBase):
-    """ Apply one of the fit modules from PYME.localization.FitFactories to each of the points in the provided
-    in inputPositions
+    """ 
+    Apply one of the fit modules from PYME.localization.FitFactories to each of
+    the points in the provided in inputPositions. 
+
+    Parameters
+    ----------
+    inputImage: PYME.IO.image.ImageStack
+        FitPoints does not do unit conversions normally done during localization
+        analysis in PYME (where we convert from raw analog-digital units [ADU]
```

Comment:
All our fitting is done in ADUs! we don't ever convert, so this is no different. We only convert to electrons in PYMEVis/Pipelineify where we generate the `nPhotons` derived variable. There are also inconsistencies between modules (in some cases A is the amplitude - i.e. max, and in others it is the sum.

Reply:
Yep, sorry, but in remFitBuf fitTask we define `self.data = cameraMaps.correctImage(...)`, which dark/flat corrects. So we probably want to change this to instruct folks to feed through `processing.FlatfieldAndDarkCorrect` first.